components/api_server/tests/routes/test_measurement.py

```diff
@@ -144,22 +144,53 @@ def test_set_status_also_sets_status_end_date_if_status_has_a_desired_response_t
         self.assertEqual(
             {
                 "description": "John Doe changed the status of 'entity title/foo/None' from '' to 'false_positive' "
-                f"and changed the status end date to '{deadline}'.",
+                f"and changed the status end date from 'None' to '{deadline}'.",
```

Comment:
Why do we write a literal `None` here, but not on lines 164, 179 and 193?

Reply:
The other three tests don't change the status end date. I guess you were comparing line 147 with 164, 179 and 193, but you should compare line 146 with 164, 179 and 193. Then there is no inconsistency.